### PR TITLE
Membership page

### PR DESCRIPTION
**Pull request by rantahar**
First draft of the membership page, describing how to become a member and some benefits of doing so.

### CODE REVIEW

File: content/about/membership.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   means we represent a larger percentage of academic researchers. We can also
+   draw on our members experiences to support our case.
+
+ - We run events that are useful to our members, such as seminars, workshops
+   and an annual unconference. Your membership fee helps make these events
+   possible.
+
+   In-person events may have a participation fee and members get discounts.
+
+ - An association is owned by it's members. As a member you can participate in
```
Comment (bast) [inline content/about/membership.md:23]:
```suggestion
 - An association is owned by its members. As a member you can participate in
```

**Comment by rantahar** [comment]
* Mailing list link is to a google form. Is there an easy (and cheap) way of managing mailing lists?

**Comment by rantahar** [comment]
It must be possible to unsubscribe

**Review by lucaferranti — APPROVED** [review]
spin-off: should we also tweet about this if/when it's ready?

File: content/about/membership.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+ - An association is owned by its members. As a member you can participate in
+   the associations annual meeting and vote on the direction the association
+   is evolving.
+
+
+### How Do I Sign Up?
+
+To become a member you need to
+ - register by filling out the membership form (https://forms.gle/FXdTv2mtBn53gHxx9).
```
Comment (lucaferranti) [inline content/about/membership.md:31]: don't know if it's intentional, but it's not rendered as link

File: content/about/membership.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+The membership fee needs to be paid after each annual meeting. To be exact,
+the membership fee expires at the end of October each year.
+
+
+### Mailing List
+
+Would you rather just follow our activities?
+ - Join the converstation on the [CodeRefinery chat](https://coderefinery.zulipchat.com) (#nordic-rse stream).
+ - If you just want to hear about our upcoming events a couple of times a year,
+   sign up to our mailing list. (https://forms.gle/DYzJn14jQCknYjez7)
```
Comment (lucaferranti) [inline content/about/membership.md:43]: don't know if it's intentional, but it's not rendered as link

File: templates/base.html [hunk 4]
```diff
@@ -75,6 +75,7 @@
               </a>
               <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                 <a class="dropdown-item" href="{{ config.base_url | safe }}/about/rse/">RSE</a>
+                <a class="dropdown-item" href="{{ config.base_url | safe }}/about/membership/">Membership</a>
```
Comment (lucaferranti) [inline templates/base.html:78]: maybe call it "How to join"?

File: templates/base.html [hunk 5]
```diff
@@ -90,7 +91,6 @@
               </a>
               <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                 <a class="dropdown-item" href="{{ config.base_url | safe }}/communities/map/">Map</a>
-                <a class="dropdown-item" href="{{ config.base_url | safe }}/communities/members/">Members</a>
```
Comment (lucaferranti) [inline templates/base.html:93]: should the members.md file be deleted since it's not reachable anymore?

File: content/about/membership.md (hunk 2 above)
Comment (rantahar) [reply content/about/membership.md]: Not intentiional, fixed

File: templates/base.html (hunk 4 above)
Comment (rantahar) [reply templates/base.html:78]: It also tells you what membership means and what the benefits are. Maybe the "become a sponsor" link could be replaced by "how to join" and point to /about/membership?

File: templates/base.html (hunk 5 above)
Comment (rantahar) [reply templates/base.html:93]: I intentionally did not in this PR, but I don't mind if you would like me to remove it here.

File: templates/base.html (hunk 4 above)
Comment (lucaferranti) [reply templates/base.html:78]: ok I think "membership" is a good name then, yes I agree the become a sponsor in the purple box should be replaced

Comment (rantahar) [reply templates/base.html:78]: Replaced it on the front page

**Pull request merged**